contrib/start_minitest

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 echo "Preparing application..."
-bundle exec rake dev:bootstrap[old_test_suite] RAILS_ENV=test
+bundle exec rake dev:bootstrap[old_test_suite] RAILS_ENV=test &> /dev/null
```

Comment:
Sounds like a bad idea. What if this goes wrong?

Reply:
Agreed, removed!